packages/ocaml-planet/ocaml-planet.0.1/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "sk826@cl.cam.ac.uk"
+authors: "KC Sivaramakrishnan"
+homepage: "https://github.com/kayceesrk/ocaml-planet"
+bug-reports: "https://github.com/kayceesrk/ocaml-planet/issues"
+license: "ISC"
+dev-repo: "https://github.com/kayceesrk/ocaml-planet/irmin.git"
```

Comment:
Yes. :-) Thanks. Fixed it. There is a missing dependency (lwt) in the opam file, causing builds to fail. I'll resubmit the pull request.